src/bot/constaints/callback_data.py

```diff
@@ -0,0 +1,6 @@
+VIEW = "view_tasks"
```

Comment:
Что содержит, так и назови:
```suggestion
VIEW_TASKS = "view_tasks"
```
В дальнейшем у нас будет много подобных каллбаков.